libpod/container_api.go

```diff
@@ -753,3 +753,8 @@ func (c *Container) RestartWithTimeout(ctx context.Context, timeout uint) (err e
 
 	return c.start()
 }
+
+// SetExitCommand sets the command to be executed by conmon when the container exits
+func (c *Container) SetExitCommand(command []string) {
```

Comment:
Can't do this, we never overwrite config in the DB

Reply:
Removed that was cruft from a previous attempt.